pages/work/enigma.vue

```diff
@@ -0,0 +1,147 @@
+          to candidates to ascertain their level of competency with key
+          cybersecurity concepts and industry best practices. Once a candidate
+          has completed an assessment through the Enigma platform, employers can
+          review their responses and overall performance and then use these
+          findings to better evaluate the candidate’s overall application.
+        </p>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h2>Challenge</h2>
+
+        <p>
+          The Enigma assessment application was in a very raw proof of concept
+          state when Ship Shape was brought in to act as the development team
+          that would create a production-ready product. Enigma already had
+          customers signed up and ready to start utilizing the application for
+          their recruiting needs, but still had a substantial roadmap of
+          features in need of implementation, including the full assessment
+          workflow and administration-related tasks. Ship Shape’s challenge was
+          to implement the crucial features necessary to launch Enigma and meet
+          customers’ various requests.
+        </p>
+
+        <h2>Solution</h2>
+
+        <h3>Next.js</h3>
+
+        <p>
+          The development team made use of
+          <a href="/https://nextjs.org/">Next.js</a> to prioritize quick loading
+          times on the client side of this application. The server-side
+          rendering capabilities of Next.js allows the Enigma application to
+          retrieve web pages and any corresponding data in one go, eliminating
+          the need for multiple network requests and subsequent delays in data
+          population.
+        </p>
+
+        <h3>Shepherd Tour</h3>
+
+        <p>
+          The Ship Shape team also had the opportunity to integrate one of our
+          own open-source projects,
+          <a href="https://shepherdjs.dev/docs/">Shepherd</a>, into the Enigma
+          platform. The Shepherd library offers a way to guide Enigma users
+          through a tour of the application, highlighting the key features users
+          should familiarize themselves with in order to use the app at its full
+          potential.
+        </p>
+
+        <h3>Cypress Testing</h3>
+
+        <p>
+          The development team also incorporated
+          <a href="https://www.cypress.io/">Cypress</a>, an integration testing
+          framework, into the Enigma application. This end-to-end testing
+          technology allowed the development team to deliver features to users
+          with confidence that critical tasks aren’t disrupted by bugs.
+        </p>
+
+        <h3>Key Application Features</h3>
+
+        <p>
+          In order to successfully launch this application, the Enigma and Ship
+          Shape development teams focused on implementing the key features that
+          are central to making Enigma the unique web application that it is.
```

Comment:
Some rewording ideas:

In order to successfully launch this application, the Enigma and Ship Shape development teams focused on implementing the following key features that make Enigma a unique assessment tool that meets its niche clients' needs.